F/finufft/build_tarballs.jl

```diff
@@ -3,19 +3,20 @@
 using BinaryBuilder, Pkg
 
 name = "finufft"
-version = v"1.1.2"
+version = v"2.0.2"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/flatironinstitute/finufft.git", "5b92daf4244c92844dd2640ef80457c030e1bf25")
+    ArchiveSource("https://github.com/flatironinstitute/finufft/archive/v2.0.2.zip", "fe83dbc22a1d13799e9db48568d8417d3028e7b831800f2a626ea19b4f0461fe")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/finufft/
+cd $WORKSPACE/srcdir/finufft-2.0.2/
```

Comment:
```suggestion
cd $WORKSPACE/srcdir/finufft*/
```
This way you don't need to update this line for new versions or if you need to switch back to a `GitSource`

Reply:
Great, thank you very much for this helpful addition!